package.json

```diff
@@ -4,18 +4,23 @@
   "version": "0.0.0",
   "description": "Minimalistic browser-based Solidity IDE",
   "scripts": {
-    "browser-test": "nightwatch --config nightwatch.js --env local",
-    "browser-test-remote-chrome": "nightwatch --config nightwatch.js --env chrome",
-    "browser-test-remote-firefox": "nightwatch --config nightwatch.js --env default",
-    "browser-test-remote-ie": "nightwatch --config nightwatch.js --env ie",
-    "browser-test-remote-safari": "nightwatch --config nightwatch.js --env safari",
-    "browser-test-remote-parallel": "nightwatch --config nightwatch.js --env safari,ie,default,chrome",
     "build": "mkdir build; browserify src/index.js -g yo-yoify -o build/app.js -t [ babelify ]; babel --plugins=transform-es2015-template-literals build/app.js --out-file build/app.js",
+    "test": "standard; babel src --out-dir babelify-src; node test/index.js",
+    "test-browser": "npm-run-all -lpr _selenium _http-serve browser-test",
+    "test-sauce": "env-cmd .env.local node sc",
     "csslint": "csslint --ignore=order-alphabetical --errors='errors,duplicate-properties,empty-rules' --exclude-list='assets/css/font-awesome.min.css' assets/css/",
-    "downloadsolc": "rm soljson.js; wget https://ethereum.github.io/solc-bin/soljson.js",
     "lint": "standard",
-    "serve": "http-server .",
-    "test": "standard; babel src --out-dir babelify-src; node test/index.js"
+    "browser-test": "nightwatch --config nightwatch.js --env local",
+    "browser-test-remote-chrome": "env-cmd .env.local nightwatch --config nightwatch.js --env chrome",
+    "browser-test-remote-firefox": "env-cmd .env.local nightwatch --config nightwatch.js --env default",
+    "browser-test-remote-ie": "env-cmd .env.local nightwatch --config nightwatch.js --env ie",
+    "browser-test-remote-safari": "env-cmd .env.local nightwatch --config nightwatch.js --env safari",
+    "browser-test-remote-parallel": "env-cmd .env.local nightwatch --config nightwatch.js --env safari,ie,default,chrome",
+    "_http-serve": "execr --silent http-server .",
```

Comment:
1. oh yeah, sry - in the process of changing and trying to automate more stuff, i renamed tasks to remind me what they do (e.g. serve runs `http-serve`) and also to not confuse myself with all these npm scripts tasks, to underscore (`_`) those, which are meant to be "private" in the sense that they are usually only called by other tasks.

2. I'm using `execr` to silence the logging, because the task `npm run test-browser` is supposed to:
  * start a local selenium server
  * start a local static http server that serves the app
  * start the saucelabs remote testing task

...the terminal in which it is started will log ALL output from all three tasks to the same terminal prefixed with their npm nam. I did not want the output of all those tasks to spam my terminal, because i'm only interested in the output from the saucelabs tests - so i'm using `execr` to silence the output. **do you think that's fine? should i change it?**